src/BenchmarkDotNet/Toolchains/MonoWasm/WasmGenerator.cs

```diff
@@ -37,6 +92,8 @@ protected override void GenerateProject(BuildPartition buildPartition, Artifacts
                     .Replace("$SDKNAME$", sdkName)
                     .Replace("$RUNTIMEPACK$", CustomRuntimePack ?? "")
                     .Replace("$TARGET$", CustomRuntimePack != null ? "PublishWithCustomRuntimePack" : "Publish")
+                    .Replace("$AOT$", runtime.Aot ? "true" : "false")
+                    .Replace("$TRIM$", runtime.Aot ? "true" : "false")
```

Comment:
`runtime.Aot` will always be false for `GenerateProjectInterpreter` method?
```suggestion
                    .Replace("$AOT$", "false")
                    .Replace("$TRIM$", "false")
```